.env

```diff
@@ -0,0 +1,3 @@
+MAKE := "/usr/bin/make"
+  GO := "/usr/local/bin/go"
+  DEP := "/usr/local/bin/dep"
```

Comment:
The whitespace here isn't significant is it?

Reply:
No, but also, this file shouldn't be checked in, my bad